example/PennFudanPed/pfp/evaluator.py

```diff
@@ -21,14 +21,20 @@ def __init__(self) -> None:
         super().__init__()
 
     @torch.no_grad()
-    def ppl(self, img: Image, index: str, **kw):
-        index = index.split("_")[-1]
+    def ppl(self, img: Image, index: t.Union[int, str], **kw):
+        if isinstance(index, str) and "_" in index and index.startswith("dataset-"):
```

Comment:
could we return the index which is set by user without any make up by starwhale.

Reply:
if the client version is not 0.3.2,  `index` is the user raw index,  make up index name is `index_with_dataset`.
see this line: https://github.com/star-whale/starwhale/pull/1553/files#diff-69e86e9f2dc216886b958a3a3791b2f7a1bfe6979dd963d0b1b13bd91fde110fR243

Reply:
line 25 is the only compatible code for all versions.